$NetBSD: patch-configure,v 1.6 2019/10/31 11:22:15 jperkin Exp $

Portability fixes.
Move the checks for $have_milter to after it is defined.

--- configure.orig	2019-10-01 17:24:08.000000000 +0000
+++ configure
@@ -18269,7 +18269,7 @@ else
 fi
 
 
- if test x$enable_libclamav_only == xyes; then
+ if test x$enable_libclamav_only = xyes; then
   BUILD_LIBCLAMAV_ONLY_TRUE=
   BUILD_LIBCLAMAV_ONLY_FALSE='#'
 else
@@ -18278,8 +18278,8 @@ else
 fi
 
 
-if test "$enable_libclamav_only" == "yes"; then
-         if test x$enable_libclamav_only == xno; then
+if test "$enable_libclamav_only" = "yes"; then
+         if test x$enable_libclamav_only = xno; then
   BUILD_CLAMONACC_TRUE=
   BUILD_CLAMONACC_FALSE='#'
 else
@@ -21224,7 +21224,7 @@ $as_echo "yes" >&6; }
 	HAVE_LIBCHECK=yes
 fi
 
-if test "X$HAVE_LIBCHECK" == "Xyes"; then
+if test "X$HAVE_LIBCHECK" = "Xyes"; then
     CHECK_CPPFLAGS=$CHECK_CFLAGS
 else
 
@@ -24689,6 +24689,12 @@ fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: checking for systemd system unit installation directory... $with_systemdsystemunitdir" >&5
 $as_echo "checking for systemd system unit installation directory... $with_systemdsystemunitdir" >&6; }
 
+# Check whether --enable-milter was given.
+if test "${enable_milter+set}" = set; then :
+  enableval=$enable_milter; have_milter=$enableval
+else
+  have_milter="no"
+fi
 
 if test "$have_milter" = "yes"; then
 
@@ -29013,10 +29019,10 @@ fi
 
 			;;
 		*)
-			if test "$enable_clamonacc" == "yes"; then
+			if test "$enable_clamonacc" = "yes"; then
 				as_fn_error $? "Clamonacc was explicitly requested, but the platform ($host_os) you are trying to build on is not currently supported for this tool." "$LINENO" 5
 			fi
-			 if test x$enable_clamonacc == xno; then
+			 if test x$enable_clamonacc = xno; then
   BUILD_CLAMONACC_TRUE=
   BUILD_CLAMONACC_FALSE='#'
 else
@@ -29027,7 +29033,7 @@ fi
                         ;;
 	esac
 else
-	 if test x$enable_clamonacc == xyes; then
+	 if test x$enable_clamonacc = xyes; then
   BUILD_CLAMONACC_TRUE=
   BUILD_CLAMONACC_FALSE='#'
 else
@@ -29296,12 +29302,6 @@ if test $ac_cv_func_setpgrp_void = yes;
 fi
 fi
 
-# Check whether --enable-milter was given.
-if test "${enable_milter+set}" = set; then :
-  enableval=$enable_milter; have_milter=$enableval
-else
-  have_milter="no"
-fi
 
 
 # Check whether --enable-id-check was given.
